src/img/icons.json

```diff
@@ -105,4 +106,4 @@
     "Sanjeet Shrimp",
     "Norman Naked Mole Rat"
   ]
-}
+}
```

Comment:
Please add an EOF new line

Same as it was before. This is most likely removed automatically by your editor, so you'll have to tweak your settings to allow them

Reply:
Done, indeed needed to change some settings in VSCode for add final newline